tests/ext/test_updater.py

```diff
@@ -73,6 +74,12 @@ def _reset(self):
         self.cb_handler_called = None
         self.test_flag = False
 
+    @pytest.fixture()
+    def file_path(self) -> str:
+        path = TEST_DATA_PATH / "test.sock"
+        yield str(path)
+        path.unlink(missing_ok=True)
+
```

Comment:
don't [pytests built-in features](https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html) suffice for this?

Reply:
where was this when I created this code grrrr

Reply:
Here https://t.me/c/1494805131/38624 🤡